### DISCUSSION TITLE
v4.2.0

### DISCUSSION THREAD

**Discussion by julien-amblard** [julien-amblard/email-misspelled#68] [General]
## What's Changed
* fix: add `types` field to `package.json` by @limonte in https://github.com/julien-amblard/email-misspelled/pull/67

## New Contributors
* @limonte made their first contribution in https://github.com/julien-amblard/email-misspelled/pull/67

**Full Changelog**: https://github.com/julien-amblard/email-misspelled/compare/v4.1.0...v4.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/julien-amblard/email-misspelled/releases/tag/v4.2.0'>v4.2.0</a>.</em>

**Comment by julien-amblard** [comment]
If you think this should have been a `4.1.1` and not a `4.2.0`, you're right. 
It seems I'm a bit tired 🤷‍♂️
Sorry for that

**Comment by limonte** [comment]
no worries @julien-amblard, 4.2.0 is fine 👍 

thanks for the prompt release 🚀
